005_Implicit_Regularized_Gaussian/implicit_priors.ipynb

```diff
@@ -0,0 +1,625 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# -*- coding: utf-8 -*-\n",
+    "#  Copyright 2023 Technical University of Denmark\n",
+    "#\n",
+    "#  Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "#  you may not use this file except in compliance with the License.\n",
+    "#  You may obtain a copy of the License at\n",
+    "#\n",
+    "#      http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "#  Unless required by applicable law or agreed to in writing, software\n",
+    "#  distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "#  See the License for the specific language governing permissions and\n",
+    "#  limitations under the License.\n",
+    "#\n",
+    "#  Authored by: Jasper Everink (DTU)\n",
+    "#  Edited by: Chao Zhang (DTU)\n",
+    "#  Reviewed by: Jakob Sauer Jørgensen (DTU)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Implicit priors with regularized Gaussian distributions\n",
+    "\n",
+    "### Jasper Everink (jmev@dtu.dk)\n",
+    "Regularization is frequently employed in variational inverse problems to enforce constraints on the problem's parameters. An example of such a constraint is sparsity, often induced through regularization techniques like lasso and total variation. In this notebook, we demonstrate how to use so-called implicit priors based on regularized Gaussian distributions from [[1]](https://iopscience.iop.org/article/10.1088/1361-6420/acf9c5) in CUQIpy.\n"
```

Comment:
Constraints have a specific meaning in this context and is not the same as assumptions. Perhaps use beliefs instead.

Reply:
Thanks @jeverink , I just replaced constraints with beliefs